docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -101,6 +101,91 @@ NOTE: The properties marked with *bold* are mandatory.
 |Defines text threshold length to enable unified diff report publishing on failure, for variable comparisons. If you want to have this report despite the size of the compared data set 0.
 |===
 
+=== Profiles
+
+A profile defines the subject of the test, i.e. it answers the question "What are specifics of the browser/device I want to run tests on?".
+
+These specifics (properties) should be written in the `profile.properties` file. Path to this file is set in `configuration.properties` file:
+
+.configuration.properties
+[source,properties]
+----
+configuration.profiles=web
+----
+
+It's allowed to create multiple profiles. This can be useful for cross-browsing or cross-device testing: there is no need of changing all the properties, just switch the profile. +
+As well as the suites, profiles are loaded one by one starting from the last one in the sequence. This allows to reuse common properties and separate specific ones.
+
+==== Profiles setup
```

Comment:
I suggest to state in this section using the warning -  that running on multiple profiles e.g desktop/chrome, desktop/firofox is not supported 
@valfirst